src/redist/targets/GeneratePKG.targets

```diff
@@ -7,8 +7,9 @@
       <SharedHostComponentId>com.microsoft.dotnet.sharedhost.$(SharedHostVersion).component.osx.x64</SharedHostComponentId>
       <HostFxrComponentId>com.microsoft.dotnet.hostfxr.$(HostFxrVersion).component.osx.x64</HostFxrComponentId>
       <SharedFrameworkComponentId>com.microsoft.dotnet.sharedframework.$(SharedFrameworkNugetName).$(MicrosoftNETCoreAppPackageVersion).component.osx.x64</SharedFrameworkComponentId>
-      <NetCoreAppTargetingPackComponentId>com.microsoft.dotnet.pack.targeting.$(MicrosoftNETCoreAppPackageVersion).component.osx.x64</NetCoreAppTargetingPackComponentId>
-      <NetStandardTargetingPackComponentId>com.microsoft.standard.pack.targeting.$(MicrosoftNETCoreAppPackageVersion).component.osx.x64</NetStandardTargetingPackComponentId>
+      <NetCoreAppTargetingPackComponentId>com.microsoft.dotnet.pack.targeting.$(NetCoreAppTargetingPackVersion).component.osx.x64</NetCoreAppTargetingPackComponentId>
+      <NetCoreAppHostPackComponentId>com.microsoft.dotnet.pack.apphost.$(NetCoreAppHostPackVersion).component.osx.x64</NetCoreAppHostPackComponentId>
+      <NetStandardTargetingPackComponentId>com.microsoft.standard.pack.targeting.$(NETStandardLibraryRefPackageVersion).component.osx.x64</NetStandardTargetingPackComponentId>
```

Comment:
Looks like we had a bug in the NetStandardTargetingPackComponentId before that this fixes? What are the consequences of that?

Reply:
Mac native installers would not have the targeting pack until this change is merged.